index.js

```diff
@@ -1,45 +1,35 @@
-const express = require('express');
-
-const app = express();
-const PORT = process.env.PORT || 3000;
-
-app.use(express.static('public'));
-app.use(express.json());
-
-app.get('/', (req, res) => {
-    res.send('yes');
-});
-
-app.post('/math', (req, res) => {
-    // res.send(req.params.expression);
-    
-    //console.log(req.body.expression);
-    // TODO maybe validate inpute here too?
-
-    //do python math here
-    const python = require('python-shell');
-    const path = require('path');
-
-    var input = req.body.expression;
-
-    console.log('my inpit: ', input);
-    result = '';
-
-    option = {
-        args: [input]
-    };
-
-
-    const pyshell = new python('calculate.py', { args: [input] });
-
-    pyshell.on('message', (msg) => {
-        //console.log(msg);
-        result = Number(msg);
-        //document.getElementById('resultInput').value = result;
-        res.status(200);
-        res.send({ answer: msg });
-    }); 
-    
-});
-
+const express = require('express');
+const python = require('python-shell');
+
+const app = express();
+const PORT = process.env.PORT || 3000;
+
+app.use(express.static('public'));
+app.use(express.json());
+
+app.get('/', (req, res) => {
+  res.send('yes');
+});
+
+app.post('/math', (req, res) => {
+  // do python math here
+  // const python = require('python-shell');
+  const input = req.body.expression;
+  // console.log('my inpit: ', input);
+
+  const option = {
+    args: [input],
+  };
+
+  // TODO change to always listening python shell instead of creating and deleting everytime
+  /* eslint-disable-next-line */
+  const pyshell = new python('calculate.py', option);
```

Comment:
just to make sure, are you sure this shell isn't killed everytime?

Reply:
I am working on this on a seperate branch and will make a seperate PR once that's ready to not overload and put too much on this PR